todo_app/data/trello_items.py

```diff
@@ -0,0 +1,104 @@
+from requests.exceptions import RequestException
+
+from todo_app.helpers.trello_helpers import trello_api_request, get_list_id_from_name
+from todo_app.models.data.item import Item
+
+
+valid_item_status_data = {
+    'To Do': {
+        'theme': 'secondary',
+    },
+    'Doing': {
+        'theme': 'info',
+    },
+    'Done': {
+        'theme': 'success',
+    },
+    'Abandoned': {
+        'theme': 'danger',
+    },
+}
+
+
+def get_items():
+    """
+    Fetches all saved items from the board.
+
+    Returns:
+        list: The list of saved items.
+    """
+    try:
+        items = []
+        for status in valid_item_status_data:
+            list_id = get_list_id_from_name(list_name=status)
+            items_in_list = map(Item.from_trello_card, trello_api_request("GET", f"/lists/{list_id}/cards").json())
+            items.extend(items_in_list)
+        return items
+    except RequestException:
+        return None
```

Comment:
Handling errors like this usually just ends up making things harder to debug - this doesn't usefully recover from not being able to connect to Trello, it just means the error will happen somewhere else

Reply:
My idea with this was to be able to differentiate between an empty list of tasks and a nonexistent list - have tried to clarify this in the index.html. Perhaps there is a better way of doing this though that would involve handling the exception at a different point?